### ISSUE TITLE
FileNotFoundException on resource creation

### ISSUE THREAD

**Issue by ibnnajjaar** [filamentphp/filament#3912]
### Package

filament/filament

### Package Version

v2.15.46

### Laravel Version

v9.28.0

### PHP Version

PHP 8.1.7

### Problem description

When running "php artisan make:filament-resource User" FileNotFoundException is thrown.

![image](https://user-images.githubusercontent.com/7188633/189203610-0cdaa2f1-f94b-4575-bba7-b01e02eaef4d.png)

### Expected behavior

Generate UserResource file

### Steps to reproduce

Install fresh laravel project,
Install filament
Do setup
run "php artisan make:filament-resource User"

### Reproduction repository

https://github.com/ibnnajjaar/filamentphpissue

### Relevant log output

```shell
php artisan make:filament-resource User

   Illuminate\Contracts\Filesystem\FileNotFoundException

  File does not exist at path C:\www\office\www\filamentphpissue\vendor\filament\filament\src\Commands\MakeResourceCommand.php/../stubs/Resource.stub.

  at C:\www\office\www\filamentphpissue\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:59
     55▕         if ($this->isFile($path)) {
     56▕             return $lock ? $this->sharedGet($path) : file_get_contents($path);
     57▕         }
     58▕
  ➜  59▕         throw new FileNotFoundException("File does not exist at path {$path}.");
     60▕     }
     61▕
     62▕     /**
     63▕      * Get contents of a file with shared access.

  1   C:\www\office\www\filamentphpissue\vendor\filament\support\src\Commands\Concerns\CanManipulateFiles.php:32
      Illuminate\Filesystem\Filesystem::get("C:\www\office\www\filamentphpissue\vendor\filament\filament\src\Commands\MakeResourceCommand.php/../stubs/Resource.stub")

  2   C:\www\office\www\filamentphpissue\vendor\filament\filament\src\Commands\MakeResourceCommand.php:157
      Filament\Commands\MakeResourceCommand::copyStubToApp("Resource", "C:\www\office\www\filamentphpissue\app\Filament/Resources/UserResource.php")
```

**Comment by danharrin** [comment]
Hmm, looks like a Windows only issue. Please check the latest release, let me know if it doesnt fix.

**Comment by yainspan** [comment]
So I'm experiencing this issue now, with creating panels.
I'm on macOS Sonoma, did a clean install of Filament.

```
 Illuminate\Contracts\Filesystem\FileNotFoundException 

  File does not exist at path /Users/user/projects/project/vendor/filament/filament/src/../stubs/DefaultPanelProvider.stub.

  at vendor/laravel/framework/src/Illuminate/Filesystem/Filesystem.php:59
     55▕         if ($this->isFile($path)) {
     56▕             return $lock ? $this->sharedGet($path) : file_get_contents($path);
     57▕         }
     58▕ 
  ➜  59▕         throw new FileNotFoundException("File does not exist at path {$path}.");
     60▕     }
     61▕ 
     62▕     /**
     63▕      * Get the contents of a file as decoded JSON.

      +15 vendor frames 

  16  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

Looking in the vendor code, it seems that the stub exists at `filament/support/stubs`, but it's looking in `filament/filament/stubs`.

**Comment by danharrin** [comment]
Should be fixed by c7d3fba
